workflow-scripts/sweep-dependabot-prs.py

```
@@ -0,0 +1,206 @@
+DEFAULT_LABEL = "automerge"
+DEFAULT_OWNER = "cuioss"
+DEFAULT_AUTHOR = "app/dependabot"
+
+# GitHub's own readiness verdict. Anything else means a required check is
+# missing, failing, or the PR is blocked -- never force it.
+READY_MERGE_STATE = "CLEAN"
+
+
+def run_gh(args: list[str]) -> subprocess.CompletedProcess[str]:
+    """Run a gh CLI command without raising on failure."""
+    return subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)